README.md

```diff
@@ -34,6 +34,8 @@ We welcome community contributions to the Open Model Zoo repository. If you have
 * In case of a larger feature, provide a relevant demo.
 * Submit a pull request at https://github.com/opencv/open_model_zoo/pulls
 
+Additional information about contributing your model to Open Model Zoo you can find [here](CONTRIBUTING.md).
```

Comment:
The same as for `Detailed structure and headers naming convention you can learn from any other model`.